### PR TITLE
Issue 13 wildcard mask

### LINKED ISSUE #13: Feature request: ipv4#wildcard_mask method
Would you consider a PR to add a wildcard_mask method to an IPv4 address? I would be happy to submit.

### PR DESCRIPTION
**Pull request by bwks**
Added `wildcard_mask` method.

Resolves #13

### CODE REVIEW

**Comment by bwks** [comment]
I will also update the README

File: src/ipaddress/ipv4.cr [hunk 1]
```diff
@@ -414,6 +414,20 @@ module IPAddress
       @prefix = Prefix32.parse_netmask addr
     end
 
+    # Returns the wildcard mask as a String
+    #
+    # ```
+    # ip = IPAddress.new "172.16.100.4/22"
+    # ip.wildcard_mask # => "0.0.3.255"
+    # ```
+    def wildcard_mask : String
```
Comment (Sija) [inline src/ipaddress/ipv4.cr:423]: I'd move it into the `Prefix32` struct, `Prefix128` for instance doesn't support it.

**Review by Sija — CHANGES_REQUESTED** [review]

**Comment by Sija** [comment]
@bwks I've merged #15, in which I moved the method into the `Prefix32` struct, so you can use it like:

```cr
ip = IPAddress.new "172.16.100.4/22"
if ip.is_a?(IPAddress::IPv4)
  ip.prefix.wildcard_mask # => "0.0.3.255"
end
```

See if that resolves your issue please and if yes, I'll release a new shard version.

**Comment by bwks** [comment]
Hey @Sija thank you very much for this. I am sorry I didn't get back to you sooner. Work has been flat out in the last couple of days and was going to come back to this on the weekend. I noticed that there is already a `Prefix32#hostmask` method when I was reading through the code and I believe it does the same thing.

https://github.com/Sija/ipaddress.cr/blob/9b16a958963b566353c88b94c1674c2764e3f3d3/src/ipaddress/prefix.cr#L179
```crystal
    # The hostmask is the contrary of the subnet mask,
    # as it shows the bits that can change within the
    # hosts.
    #
    # ```
    # prefix = IPAddress::Prefix32.new 24
    # prefix.hostmask # => "0.0.0.255"
    # ```
    def hostmask : String
      octets.join('.') { |i| ~i.to_u8 }
    end
  end
```

Please correct me if im wrong, but maybe you already implemented it and forgot? This new method may be unnecessary.

**Comment by Sija** [comment]
Haha, you're so right 😅

**Comment by bwks** [comment]
🤣 
Thank so much anyway for the support ❤. I learned a lot just attempting this.

**Pull request closed without merging**